Dockerfile

```diff
@@ -1,16 +1,17 @@
-FROM ruby:2.6
+FROM registry.access.redhat.com/ubi8/ruby-26
 
-WORKDIR /app
+USER root
```

Comment:
Is this advised somewhere for UBI images?

Reply:
No, the base image comes as USER 1001 - I only move to root to copy the source into `/tmp/src` , which is the location used by `s2i/assemble` - after that, I go back to 1001

Reply:
```

mv: cannot move '/tmp/src/Gemfile' to './Gemfile': Permission denied
mv: cannot move '/tmp/src/Gemfile.local.example' to './Gemfile.local.example': Permission denied
mv: cannot move '/tmp/src/Gemfile.lock' to './Gemfile.lock': Permission denied
```
happens if removed

Reply:
Using (or not using) root has security implications which don't matter in devel. Red Hat has taken the position that we will ship non-root images because of security, but they are meant to be used primarily in prod. Ideally, we could build the image as root and then run it via the docker-compose under whatever the $UID/$GID env variables are set to so file access is seamless in a development environment.

Reply:
Again - this doesn't build the image as root nor makes it the default - it's the same as we do in prod. I revert to 1001 a few lines after: 
  - 1. Set root
  - 2. Copy source to /tmp/src - you *must* be root to do it
  - 3. Allow user 1001 to read it
  - 4. Set user back to 1001